templates/strings/objc-m.stencil

```diff
@@ -0,0 +1,80 @@
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+{% if tables.count > 0 %}
+
+#import "Localizable.h"
```

Comment:
This should be parameterizable. For example: `{{ param.headerName|default:"Localizable.h" }}`
Not sure if it should contain the `.h` extension or not though 🤷‍♂️ 

Don't forget to add the documentation in the template docs for this, and a test variation to test the import.